tests/func/parsing/test_interpolated_entry.py

```diff
@@ -296,22 +298,55 @@ def test_cmd_dict(tmp_dir, dvc, bool_config, list_config):
         bool_resolved = " --bool --no-bool-false"
 
     if list_config is None or list_config == "nargs":
-        list_resolved = " --list 1 2 'foo'"
+        list_resolved = ' --list 1 2 "foo"'
     else:
-        list_resolved = " --list 1 --list 2 --list 'foo'"
+        list_resolved = ' --list 1 --list 2 --list "foo"'
 
     assert_stage_equal(
         resolver.resolve(),
         {
             "stages": {
                 "stage1": {
                     "cmd": "python script.py"
-                    " --foo 'foo' --bar 2"
-                    " --string 'spaced string'"
+                    ' --foo "foo" --bar 2'
+                    ' --string "spaced string"'
                     f"{bool_resolved}"
                     f"{list_resolved}"
-                    " --nested.foo 'foo'"
+                    ' --nested.foo "foo"'
                 }
             }
         },
     )
+
+
+def test_str_interpolation_added_quotes(tmp_dir, dvc):
```

Comment:
Please don't add e2e tests here. Besides, it does not make sense to have this test at all.

Reply:
Apologies, was my lazy way for reproducing the error on a windows machine